.github/workflows/aws_openshift_rosa_hcp_single_region_tests.yml

```diff
@@ -462,6 +462,12 @@ jobs:
                     kubectl -n "$TEST_NAMESPACE" logs "$pod_name";
                   done
 
+            - name: Prevent other runs for renovate
+              if: ${{ success() && env.IS_RENOVATE_PR == 'true' }}
+              env:
+                  GH_TOKEN: ${{ github.token }}
+              uses: ./.github/actions/internal-apply-skip-label
+
```

Comment:
the problem of calling it in matrix build is that one could succeed the other one fail and potentially hiding a problem with future runs as we're skipping it.
The extra comments in a PR don't matter that would be caused of it, don't have to overcomplicate it on first iteration.
Just saying that being called in a matrix may hide issues.

Reply:
quick fix, could just be a separate job worst case, after cleanup or in between clean and integration, how you like.
The if would skip the execution on normal executions and only affects Renovate runs.